src/encoded/static/components/top_hits/hits/base.js

```diff
@@ -0,0 +1,321 @@
+import _ from 'underscore';
+import {
+    MAX_WORDS_SHORT,
+    SCIENTIFIC_NAME,
+    TERM_NAME,
+    WHITESPACE,
+} from '../constants';
+import { fieldToComponent } from './parts';
+
+
+/**
+* Shortens a sentence to maxWords.
+* @param {string} value - Sentence to trim.
+* @param {number} maxWords - Max number of words to return.
+* @return {string} Trimmed sentence.
+*/
+export const maybeTrimByWord = (value, maxWords) => {
+    const splitValue = value.split(WHITESPACE);
+    if (splitValue.length > maxWords) {
+        return `${splitValue.slice(0, maxWords).join(WHITESPACE)}...`;
+    }
+    return value;
+};
```

Comment:
Not sure if it’s appropriate here, but we have the `truncateString` function in globals.js that truncates a string to a number of characters, back to the previous word break before that character count. Embarrassingly, even _I_ didn’t comment that function.

Reply:
Ah all types of useful functions in there.